.github/workflows/build.yml

```diff
@@ -464,9 +464,13 @@ jobs:
         with:
           components: clippy
       # - we want to be free of any warnings, so deny them
-      # - disable incompatible_msrv as it does not understand that we apply our
+      # - allow clippy::disallowed_types as workaround since disallowed_types configured in .clippy.toml
+      #   is only intended for main rustls crate
+      # - allow clippy::incompatible_msrv as it does not understand that we apply our
       #   MSRV to the just the core crate.
-      - run: ./admin/clippy -- --deny warnings --allow clippy::incompatible_msrv
+      - run: ./admin/clippy -- --deny warnings --allow clippy::disallowed_types --allow clippy::incompatible_msrv
```

Comment:
Why do we need these changes?

Reply:
As I tried to explain in the comments, this is a workaround since I couldn't figure out how to configure the `disallowed-types` *only* for the main rustls crate. This is definitely ugly but I couldn't figure out another solution. Any ideas or pointers would be much appreciated on this.

We _could_ also remove this in favor of configuring `disallowed-types = "allow"` in Config.toml for all other crates. This would mean quite a few more changes, which I would not likely favor from my side.

I would love to see this supported on the Rust Cargo side.

Reply:
So sorry I am not sure I understand your comment 100%. I just pushed an update to add a TODO comment. I am thinking it may be easiest if you can contribute the required verbiage, like cpu did for some other comments, and put your name in the commit co-authored-by lines.